asar-teensy-firmware/TestScripts/XBEE/Xbee_Write/Xbee_Write.ino

```diff
@@ -0,0 +1,26 @@
+#include "Wire.h"
+
+#define XBEE Serial1
```

Comment:
Should this test be using your class?

Reply:
No, I was just trying to use this to write and test that the xbee read is working. Have not put effort into this code yet.

Reply:
Ohkay, if this is just a test, that's fine.